content-src/asrouter/templates/EOYSnippet/EOYSnippet.schema.json

```diff
@@ -0,0 +1,136 @@
+{
+  "title": "EOYSnippet",
+  "description": "Fundraising Snippet",
+  "version": "1.0.0",
+  "type": "object",
+  "definitions": {
+    "plainText": {
+      "description": "Plain text (no HTML allowed)",
+      "type": "string"
+    },
+    "richText": {
+      "description": "Text with HTML subset allowed: i, b, u, strong, em, br",
+      "type": "string"
+    },
+    "link_url": {
+      "description": "Target for links or buttons",
+      "type": "string",
+      "format": "uri"
+    }
+  },
+  "properties": {
+    "donation_form_url": {
+      "type": "string",
+      "description": "Url to the donation form."
+    },
+    "currency_code": {
+      "type": "string",
+      "description": "The code for the currency. Examle gbp, cad, usd."
+    },
+    "locale": {
+      "type": "string",
+      "description": "String for the locale code."
+    },
+    "text": {
+      "allOf": [
+        {"$ref": "#/definitions/richText"},
+        {"description": "Main body text of snippet. HTML subset allowed: i, b, u, strong, em, br"}
+      ]
+    },
+    "text_color": {
+      "type": "string",
+      "description": "Modify the text message color"
+    },
+    "background_color": {
+      "type": "string",
+      "description": "Snippet background color."
+    },
+    "highlight_color": {
+      "type": "string",
+      "description": "Paragraph em highlight color."
+    },
+    "donation_amount_first": {
+      "type": "number",
+      "description": "First button amount."
+    },
+    "donation_amount_second": {
+      "type": "number",
+      "description": "Second button amount."
+    },
+    "donation_amount_third": {
+      "type": "number",
+      "description": "Third button amount."
+    },
+    "donation_amount_fourth": {
+      "type": "number",
+      "description": "Fourth button amount."
+    },
+    "selected_button": {
+      "type": "number",
+      "description": "Default donation_amount_second. Donation amount button that's selected by default."
+    },
+    "icon": {
+      "type": "string",
+      "description": "Snippet icon. 64x64px. SVG or PNG preferred."
+    },
+    "title_icon": {
+      "type": "string",
+      "description": "Small icon that shows up before the title / text. 16x16px. SVG or PNG preferred. Grayscale."
+    },
+    "button_label": {
+      "allOf": [
+        {"$ref": "#/definitions/plainText"},
+        {"description": "Text for a button next to main snippet text that links to button_url. Requires button_url."}
+      ]
+    },
+    "button_color": {
+      "type": "string",
+      "description": "The text color of the button. Valid CSS color."
+    },
+    "button_background_color": {
+      "type": "string",
+      "description": "The background color of the button. Valid CSS color."
+    },
+    "block_button_text": {
```

Comment:
Must default to `Remove this`. I know we discussed this before but I don't understand how that's different from all other default values.